README.md

```diff
@@ -60,3 +64,68 @@ Results in this JSON Schema:
   }
 }
 ```
+# flags
```

Comment:
This should be `##`, and name it "Configurable behaviour". Then describe how creating a new reflector lets you alter how it behaves. Without this context, this makes no sense to a new user.